log/log.go

```diff
@@ -58,3 +60,32 @@ type LoggerFunc func(...interface{}) error
 func (f LoggerFunc) Log(keyvals ...interface{}) error {
 	return f(keyvals...)
 }
+
+// SwapLogger wraps another logger that may be safely replaced while other
+// goroutines use the SwapLogger concurrently.
```

Comment:
Would you mind adding a comment about intended usage? e.g. "One application of a SwapLogger would be a package global logger that can be changed by importers."